### ISSUE TITLE
[Donations] Make it work with Paypal and revisit the designs.

### ISSUE THREAD

**Issue by roughprada** [your-digital-rights/yourdigitalrights.org#145]
![Screenshot 2020-06-03 at 10 16 17](https://user-images.githubusercontent.com/5957010/83619160-5dba2300-a583-11ea-8d29-6cab2a7dae37.png)

**Blocked until we have the Charity body and a bank account to support it.**

**Comment by yoavaviram** [comment]
Not with Paypal please. They are evil. 
How about: https://liberapay.com/

**Comment by roughprada** [comment]
Yeah, sounds good. I mean paypal as any other money transfer company.
Once we have the bank account for the charity we will be able to activate this.
